login-workflow/example2/src/screens/new-architecture-test-screens/SetPasswordScreen.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useCallback, useEffect, useRef } from 'react';
+import {
+    defaultPasswordRequirements,
+    SetPassword,
+    useLanguageLocale,
+    AuthUIContextProvider,
+    useSecurityActions,
+    BrandedCardContainer,
```

Comment:
This file looks like our original, why are we using branded card container, t function on lang. etc. @daileytj  what are the plans here? Will this be refactored later?

Reply:
Anything that is used should be used from the new-architecture folder. If there are items we are using that need migrated then migrate them. In the case of BrandedCardContainer, we'll need to have that replaced with the new WorkflowCard components

Reply:
Done. We are using everything from new-architecture and replaced BrandedCardContainer with WorkflowCard. Please review.